recipes/nibabel/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: nibabel
+  version: "2.0.2"
+
+source:
+  fn: nibabel-2.0.2.tar.gz
+  url: https://pypi.python.org/packages/source/n/nibabel/nibabel-2.0.2.tar.gz
+  md5: fa89163149e35f52e237b2cb56b1c606
```

Comment:
Some people like to use jinja templates for things that get repeated often like version numbers. It makes it much easier to update. Here is an [example]( https://github.com/conda-forge/megaman-feedstock/blob/master/recipe/meta.yaml#L1-L9 ).